ratis-server-api/src/main/java/org/apache/ratis/server/raftlog/RaftLogSequentialOps.java

```
@@ -132,21 +133,34 @@ default CompletableFuture<Long> appendEntry(LogEntryProto entry, TransactionCont
   /**
    * The same as append(Arrays.asList(entries)).
    *
-   * @deprecated use {@link #append(List)}
+   * @deprecated use {@link #append(List, ReferenceCountedObject)}
    */
   @Deprecated
   default List<CompletableFuture<Long>> append(LogEntryProto... entries) {
     return append(Arrays.asList(entries));
   }
 
+  /**
+   * @deprecated use {@link #append(List, ReferenceCountedObject)}.
+   */
+  @Deprecated
+  default List<CompletableFuture<Long>> append(List<LogEntryProto> entries) {
+    return append(entries, null);
+  }
```

This should throw `UnsupportedOperationException`.